examples/new_fitting_Tgas.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
```

Comment:
File should be named "plot_something" for ReadTheDocs to generate plots

So "plot_fitting_Tgas"

Reply:
Wow, new thing to learn. Is this some of Readthedoc 's rules, or our native setup? (if yes could you give me a reference? I want to know more about this automatic wiki setup later on)

Reply:
it's just a ReadTheDocs rule, don't waste time on investigating this ;)